### PR TITLE
For Talcommunity

### PR DESCRIPTION
**Pull request by ramy-coder**

### CODE REVIEW

**Comment by ramy-coder** [comment]
Please review and approve/give your comments to continue building the Talent community for Dev Recruit

File: types/generated/components.d.ts
Comment (chungthuang) [inline types/generated/components.d.ts:1]: This looks neat. How was it generated? @Enjoy2Live I think you will be excited to this. It ties back to your idea of merging frontend and backing code base.

File: src/api/dl-tal-community/content-types/dl-tal-community/schema.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  },
+  "options": {
+    "draftAndPublish": true
+  },
+  "pluginOptions": {},
+  "attributes": {
+    "name": {
+      "type": "string"
+    },
+    "emailID": {
```
Comment (chungthuang) [inline src/api/dl-tal-community/content-types/dl-tal-community/schema.json:17]: Does someone need to have an account with Dev Launchers' website to join the talent community? If so, we can link to their user profile https://github.com/dev-launchers/strapiv4/blob/main/src/extensions/users-permissions/content-types/user/schema.json#L22

Comment (chungthuang) [reply src/api/dl-tal-community/content-types/dl-tal-community/schema.json:17]: Discussed offline, people don't need to have an account to use this feature.

**Review by chungthuang — APPROVED** [review]

File: types/generated/components.d.ts
Comment (Enjoy2Live) [reply types/generated/components.d.ts:1]: I knew about it all along but never really told you guys as we were busy with migration. I even have an epic for it!
https://app.zenhub.com/workspaces/platform-enablement-63529f02029ee50018fe58c6/epics/Z2lkOi8vcmFwdG9yL1plbmh1YkVwaWMvMTczNTg1?state=OPEN&state=TODO&state=IN_PROGRESS&labelIds=Z2lkOi8vcmFwdG9yL1plbmh1YkxhYmVsLzUxNDgzNzM

**Pull request merged**